### DISCUSSION TITLE
Share your Custom Mount Configurations

### DISCUSSION THREAD

**Discussion by meldurson** [meldurson/MountUpRestored#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to share your mount configs that you have made so others can use them. We hope that you:
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with the different creature mounts you are sharing, followed by the configuration:

**Comment by Anuuviem** [comment]
Mount 1 is Drake. It is finnicky on the ascend and descend (goes insanely fast!). Getting the saddle and player mount point to line up is a pain but it does work. Just tap that ctrl or space key!

Mount 2 is Moder. Not as rapid as the drake, no, quite the opposite. Ascent and descent take FOREVER. But flight is smooth. I disabled stamina drain on both of them.

[Custom Mount 1]

## Custom Mount Prefab Name
# Setting type: String
# Default value: CustomMount 1
Name = Hatchling

## Path in Mount Prefab where Saddle and player Mount Point should be placed (example: For the Boar it would be 'CG/Pelvis/Spine/Spine1'
# Setting type: String
# Default value: 
Mount Path = Hatchling_mountain/Armature/Root/Spine1

## Custom Mount Stamina
# Setting type: Single
# Default value: 120
Stamina = 120

## Custom Mount Saddle Tier (1=Simple/Boar Saddle, 2=Standard/Wolf Saddle, 3=Advanced/Lox Saddle
# Setting type: Int32
# Default value: 3
# Acceptable value range: From 1 to 3
Saddle Tier = 3

## Point relative to the Mount Path that player should be mounted
# Setting type: Vector3
# Default value: {"x":0.0,"y":1.0,"z":0.0}
Mount Point = {"x":0.0,"y":3.1,"z":-0.20000000298023225}

## Point relative to the Mount Path where the saddle should be placed
# Setting type: Vector3
# Default value: {"x":0.0,"y":1.0,"z":0.0}
Saddle Position = {"x":0.0,"y":3.0,"z":-0.20000000298023225}

## Scaling of the saddle
# Setting type: Vector3
# Default value: {"x":1.0,"y":1.0,"z":1.0}
Saddle Scale = {"x":0.8,"y":0.8,"z":0.8}

## Rotation of the saddle
# Setting type: Vector3
# Default value: {"x":0.0,"y":0.0,"z":0.0}
Saddle Rotation = {"x":290.0,"y":0.0,"z":0.0}

## Stamina Drain for Flying
# Setting type: Single
# Default value: 0
Flying Stamina = 0

[Custom Mount 2]

## Custom Mount Prefab Name
# Setting type: String
# Default value: CustomMount 2
Name = Dragon

## Path in Mount Prefab where Saddle and player Mount Point should be placed (example: For the Boar it would be 'CG/Pelvis/Spine/Spine1'
# Setting type: String
# Default value: 
Mount Path = Armature/Root/Hips/Spine/Spine1

## Custom Mount Stamina
# Setting type: Single
# Default value: 120
Stamina = 500

## Custom Mount Saddle Tier (1=Simple/Boar Saddle, 2=Standard/Wolf Saddle, 3=Advanced/Lox Saddle
# Setting type: Int32
# Default value: 3
# Acceptable value range: From 1 to 3
Saddle Tier = 2

## Point relative to the Mount Path that player should be mounted
# Setting type: Vector3
# Default value: {"x":0.0,"y":1.0,"z":0.0}
Mount Point = {"x":0.0,"y":7.0,"z":-1.5}

## Point relative to the Mount Path where the saddle should be placed
# Setting type: Vector3
# Default value: {"x":0.0,"y":1.0,"z":0.0}
Saddle Position = {"x":0.0,"y":7.0,"z":-1.5}

## Scaling of the saddle
# Setting type: Vector3
# Default value: {"x":1.0,"y":1.0,"z":1.0}
Saddle Scale = {"x":1.7000000476837159,"y":1.7000000476837159,"z":1.7000000476837159}

## Rotation of the saddle
# Setting type: Vector3
# Default value: {"x":0.0,"y":0.0,"z":0.0}
Saddle Rotation = {"x":270.0,"y":0.0,"z":0.0}

## Stamina Drain for Flying
# Setting type: Single
# Default value: 0
Flying Stamina = 0
